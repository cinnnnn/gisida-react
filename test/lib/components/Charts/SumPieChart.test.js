/*

Still Not entirely defined, also not sure about JSX vs JS 

import React from 'react';
import { SumPieChart }  from '../../../../src/lib/components/Charts/SumPieChart.jsx'
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

const componentWrapper = shallow(
	<SumPieChart 
		layerId=''
		layerData={{}}
		chartSpec={{}}
		layer={{}}
	/>
);

describe('SumPieChart', () => {
	it('SumPieChart component renders correctly', () => {
		const json = toJson(componentWrapper)
		expect(json).toMatchSnapshot();
	})
});

*/