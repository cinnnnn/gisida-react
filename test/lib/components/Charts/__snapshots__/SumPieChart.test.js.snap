// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SumPieChart SumPieChart component renders correctly 1`] = `
<div
  className="pie"
>
  <PieChart
    chartHeight={225}
    chartSpacing={
      Object {
        "spacing": Array [
          10,
          100,
          10,
          0,
        ],
      }
    }
    chartWidth={325}
    dataLabelOptions={false}
    donut={50}
    legendOptions={
      Object {
        "align": "right",
        "floating": true,
        "itemStyle": Object {
          "lineHeight": 21,
        },
        "labelFormat": "{name}",
        "layout": "vertical",
        "verticalAlign": "middle",
        "x": 110,
        "y": 0,
      }
    }
    seriesData={Array []}
    showInLegend={true}
    titleOptions={false}
    tooltipOptions={
      Object {
        "backgroundColor": "rgba(255,255,255,0)",
        "borderWidth": 0,
        "formatter": [Function],
        "positioner": [Function],
        "shadow": false,
        "useHTML": true,
      }
    }
  />
</div>
`;
